x-pack/plugins/secops/public/components/timeline/body/renderers/suricata_links.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { uniq } from 'lodash/fp';
+import { suricataRulesRef } from './suricata_rules_ref';
```

Comment:
How frequently do we need to update `suricata_rules_ref'?

Reply:
Also, is this something that could one day live in ES and then be updated as-needed via the new Kibana Task Management functionality?

Reply:
What if the rules were published as a `npm` package?

Reply:
It should one day live in ES whenever Suricata rules are moved to ES. I wish that Suricata would push the refs into the alerts during the time it causes alerts but it sadly does not do that at the moment. However, if we ever had a feature to write rules and store them in ES or if we needed to turn rules on or off in ES then the rules would be in ES and I could from there generate this server side.

However, I can probably move this to an `npm` package for now as an interim step.